src/reporter.ts

```diff
@@ -91,26 +91,32 @@ export default class SauceReporter extends SummaryFormatter {
 
   logTestCase(testCaseFinished: { testCaseStartedId: string }) {
     const testCaseAttempt = this.eventDataCollector.getTestCaseAttempt(testCaseFinished.testCaseStartedId);
-    const parsed = formatterHelpers.parseTestCaseAttempt({
-      snippetBuilder: this.snippetBuilder,
-      supportCodeLibrary: this.supportCodeLibrary,
-      testCaseAttempt
-    });
+    let parsed: any;
+    try {
+      parsed = formatterHelpers.parseTestCaseAttempt({
+        snippetBuilder: this.snippetBuilder,
+        supportCodeLibrary: this.supportCodeLibrary,
+        testCaseAttempt
+      });
+    } catch (e) {
```

Comment:
Is it possible to also specifically identify the exception when a test is run more than once here, so that we can provide some guidance? (cc: @mhan83 )

Reply:
Let me check more details

Reply:
It's a `TypeError`: 
```
TypeError: Cannot read properties of undefined (reading 'line')
           at Object.parseTestCaseAttempt (/Users/michaelhan/code/saucelabs/sauce-playwright-runner/tests/fixtures/cucumber/node_modules/@cucumber/cucumber/src/formatter/helpers/test_case_attempt_parser.ts:147:9)
           at SauceReporter.logTestCase (/Users/michaelhan/code/saucelabs/sauce-playwright-runner/tests/fixtures/cucumber/node_modules/@saucelabs/cucumber-reporter/src/reporter.js:92:52)
           at EventEmitter.<anonymous> (/Users/michaelhan/code/saucelabs/sauce-playwright-runner/tests/fixtures/cucumber/node_modules/@saucelabs/cucumber-reporter/src/reporter.js:78:22)
           at EventEmitter.emit (node:events:525:35)
           at EventEmitter.emit (node:domain:489:12)
           at TestCaseRunner.run (/Users/michaelhan/code/saucelabs/sauce-playwright-runner/tests/fixtures/cucumber/node_modules/@cucumber/cucumber/src/runtime/test_case_runner.ts:231:29)
           at async Runtime.runTestCase (/Users/michaelhan/code/saucelabs/sauce-playwright-runner/tests/fixtures/cucumber/node_modules/@cucumber/cucumber/src/runtime/index.ts:91:20)
           at async Runtime.start (/Users/michaelhan/code/saucelabs/sauce-playwright-runner/tests/fixtures/cucumber/node_modules/@cucumber/cucumber/src/runtime/index.ts:121:7)
           at async runCucumber (/Users/michaelhan/code/saucelabs/sauce-playwright-runner/tests/fixtures/cucumber/node_modules/@cucumber/cucumber/src/api/run_cucumber.ts:125:19)
           at async Cli.run (/Users/michaelhan/code/saucelabs/sauce-playwright-runner/tests/fixtures/cucumber/node_modules/@cucumber/cucumber/src/cli/index.ts:78:25)
```

Is that what you're also seeing @tianfeng92?

Reply:
yep. Same error

Reply:
updated: https://app.saucelabs.com/tests/32daaf1ea7434de6b2dfb9251a90e846

Reply:
It's hard to see the warning in the output since its a bit noisy. Maybe a bit of formatting to improve readability?

Reply:
<img width="1285" alt="Screen Shot 2023-01-12 at 5 20 09 PM" src="https://user-images.githubusercontent.com/71669683/212215124-26f3a4d0-49a3-4721-be54-c7685000e5a4.png">
updated